phpBB/phpbb/db/extractor/base_extractor.php

```diff
@@ -164,13 +158,13 @@ public function init_extractor($format, $filename, $time, $download = false, $st
 
 		if ($store === true)
 		{
-			$file = $this->phpbb_root_path . 'store/' . $filename . $ext;
+			$file = sys_get_temp_dir() . '/' . $filename . $ext;
```

Comment:
I remember there being issues with the system temp dir not being available on some shared hosters in the past. Isn't the store directory the safer alternative?

Reply:
Ok, i will see
It is also used on remote_storage.php (already merged) https://github.com/phpbb/phpbb/search?utf8=%E2%9C%93&q=sys_get_temp_dir&type=